samples/archivist_story_runner.py

```diff
@@ -0,0 +1,305 @@
+"""archivist story runner takes a yaml config file listing operations to be conducted serially.
+
+It then runs the defined operations.
+"""
+
+import time
+from enum import Enum
+import argparse
+import yaml
+
+# remove warning from console
+import urllib3
```

Comment:
intersatuing - why do you need this?

Reply:
Was to remove warning, but as later comment suggests, have removed